src/foremast/awslambda/awslambda.py

```diff
@@ -181,6 +182,15 @@ def update_function_configuration(self, vpc_config):
                 Timeout=int(self.timeout),
                 MemorySize=int(self.memory),
                 VpcConfig=vpc_config)
+
+            if self.concurrency_limit:
+                self.lambda_client.put_function_concurrency(
+                    FunctionName=self.app_name,
+                    ReservedConcurrentExecutions=self.concurrency_limit
+                )
+            else:
+                self.lambda_client.delete_concurrency(FunctionName=self.app_name)
```

Comment:
Is deleting the right thing to do if no concurrently limit is provided? What would this do if an app never had a concurrency limit?

Reply:
I _think_ this will be a no-op but the docs aren't completely clear.  If we don't do this then we need to interrogate the current state to see if it is set, then remove it. That would be another request.  This seemed like the simplest solution even if we need to catch a failure.